Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -2043,7 +2043,7 @@ RefPtr<API::Navigation> WebPageProxy::loadFile(const String& fileURLString, cons
 
         weakProcess->markProcessAsRecentlyUsed();
         if (weakProcess->isLaunching())
-            weakThis->send(Messages::WebPage::LoadRequestWaitingForProcessLaunch(WTFMove(loadParameters), resourceDirectoryURL, weakThis->internals().identifier, checkAssumedReadAccessToResourceURL));
+            weakThis->send(Messages::WebPage::LoadRequestWaitingForProcessLaunch(WTFMove(loadParameters), resourceDirectoryURL, weakThis->identifier(), checkAssumedReadAccessToResourceURL));
```

Comment:
Same thing here - `weakThis` and `weakProcess` need RefPtr

Reply:
Also, why are we removing identifier from internals? `internals().identifier` seems safe.

Reply:
I'm sorry I don't get the point maybe. I'm confusing why double dereference is safe. Please fix my thinking. WebPage's internal is deferent memory block and needs to dereference twice. It cannot fit in one instruction so that it should be treaded as not safe. This is my understanding.